vowpalwabbit/interactions.cc

```diff
@@ -168,6 +210,27 @@ inline bool must_be_left_sorted(const std::vector<namespace_index>& oi)
   return false;  // 'aaa' or 'abc'
 }
 
+std::vector<std::vector<namespace_index>> expand_quadratics_wildcard_interactions(
+    bool leave_duplicate_interactions, const std::set<namespace_index>& new_example_indices)
+{
+  std::set<std::vector<namespace_index>> interactions;
+
+  for (auto it = new_example_indices.begin(); it != new_example_indices.end(); ++it)
+  {
+    auto idx1 = *it;
```

Comment:
is the copy to `idx1` and `idx2` done for readability here?

Reply:
Yep just for readability